Python/Data Structure and Algorithms/SumEvenFactors.py

```diff
@@ -22,10 +22,10 @@ def sum_even_factors( n):
 #sum_even_factors(n)=>function call
 
 #Test case 1:
-print(sum_even_factors(18))
+print("The sum of even factors is ",sum_even_factors(18))
```

Comment:
Very Nice. Reviewing your changes and then pushing another change is a good approach. I recommend before pushing your changes try to review your code 1 or 2 times. So that you can catch the Nitpicks or improper logic and improve your code

Reply:
Thank you for your advice ,I will keep that in mind !